firmware/Sources/HAL/HAL_STS/hal_sts.c

```diff
@@ -0,0 +1,118 @@
+/*********************************************************************************
+* @file           : hal_sts.c
+* @brief          : Implementation of HAL_STS
+***********************************************************************************/
+
+/**********************************************************************************/
+/*                  Include common and project definition header                  */
+/**********************************************************************************/
+#include "hal_sts.h"
+
+#include "stm32f3xx_hal.h"
```

Comment:
check with @mariuscrsn . All hal.h is needed???

Reply:
Solved, is not needed